setup.cfg

```diff
@@ -35,8 +35,9 @@ package_dir =
     = src
 packages = find:
 include_package_data = True
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*
 install_requires =
-    networkx>=2.4
+    networkx>=2.2
```

Comment:
Hi @fuenfundachtzig :wave:. In PR #27 you set the minimum required version to be `2.4`, but the last supported version of `networkx` under Python 2.7 was `2.2`. We don't want to break Python 2.7 until we can release `v0.1.0` and then drop Python 2 support (Issue #43), so do you know of something in particular that _needs_ `networkx` `v2.4`? Or was this more a recommendation?

Reply:
cc @cburton12 as well if they know.